Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem 'heroku_hatchet', "~> 7.0"
```

Comment:
Please retain that version selector; otherwise, a `bundle update` would bump to Hatchet v8, v9 etc, which is likely to change its API in a breaking fashion

Reply:
I would ask that you not lock. I'll do my best to keep things stable and I'll deprecate before breaking whenever possible. Before starting this shared test work I assumed everyone was on a reasonably recent version of hatchet, but many were several major releases behind.

Your dependencies are still locked in the Gemfile.lock, to update you have to:

```
gem install heroku_hatchet
 && bundle update heroku_hatchet
```

It won't just update itself.

Reply:
Also if one locks in `Gemfile`, I'm presuming Dependabot might not open a PR to update it, which defeats the point :-)

Reply:
> Also if one locks in Gemfile, I'm presuming Dependabot might not open a PR to update it, which defeats the point :-)


Exactly. Dependabot won't send you a PR if you've told bundler to only resolve within this range